vidify/player/web.py

```diff
@@ -0,0 +1,83 @@
+"""
+"""
+
+import logging
+from typing import Optional
+from threading import Thread
+from webbrowser import open
+from threading import Timer, Thread
+
+try:
+    import flask
+except OSError:
+    raise ImportError("Flask is not installed")
+
+from vidify import Platform, CURRENT_PLATFORM
+from vidify.player.generic import PlayerBase
+from PyQt5.QtCore import QObject
+import sys
+import os
+
+PORT = 9999
+ROOT_URL = 'http://localhost:{}/video'.format(PORT)
+DEFAULT_VIDEO_ID = "GfKs8oNP9m8"
+class WebPlayer(PlayerBase, QObject):
+    DIRECT_URL = False
+    app = None
+    currentMedia = ""
+    isPlaying = True
+    def __init__(self, vlc_args: Optional[str] = None) -> None:
+        super().__init__()
+        print(__name__)
+        self.app = flask.Flask("flask_web_server", template_folder="vidify/player/templates", static_folder="vidify/player/static")
+        self.__add_endpoints()
+        app_thread = Thread(target=self.__runFlaskWebServer)
+        app_thread.daemon = True
+        Timer(3, open, args=[ROOT_URL]).start()
```

Comment:
Not sure what this is for

Reply:
If you mean this line
`Timer(3, open, args=[ROOT_URL]).start()` 
this opens a new web page in your default browser after 3 seconds on `ROUTE_URL`. The other stuff is for creating the flask app, registering the endpoints and creating a Thread for the server to run in.

Reply:
Oh, we don't really have to wait. I thought I would just leave some time for the user to see the GUI but don't think that's necessary so I can remove it.

Reply:
Removed the wait :)